NEWS.md

```diff
@@ -24,4 +24,4 @@
 
 # pacta.portfolio.allocate 0.0.1
 
-* Added a `NEWS.md` file to track changes to the package.
+* Added a `NEWS.md` file to track changes to the package. This package is a hard-fork of [pacta.portfolio.analysis](https://github.com/RMI-PACTA/pacta.portfolio.analysis/commit/3cbc3c1f528e8ec34fbcdd37aa98a6aae330a16d) at commmit `3cbc3c1`.
```

Comment:
This comment suggests to me that `pacta.portfolio.allocate` is meant to fully capture the functionality of `pacta.portfolio.analysis`, but some of that functionality has been extracted into other repos. It may be the case (don't remember exactly) that this repo is just a fork, and all of the functionality that has been extracted into other repos is still duplicated in this repo, but that is/was not the near term goal/intent.

Reply:
Hmmm ok. Would you prefer I remove the comment? 

I thought it would be helpful in the NEWS to point back to the original package, in case we need to do any commit sleuthing.

Reply:
maybe something like "This new R package is intended to replace the allocation related functionality that used to be found in the `pacta.portfolio.analysis` package."?